gravitee-am-cypress/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "gravitee-am-cypress",
+    "version": "3.16.0-SNAPSHOT",
+    "private": true,
+    "description": "Gravitee.io AM - Cypress",
+    "scripts": {
+        "test": "cypress open",
+        "test:bulk": "cypress run --config-file cypress/integration/bulk/cypress-config.json",
+        "test:rest-api": "cypress run --config-file cypress/integration/rest-api/cypress-config.json",
+        "test:e2e": "cypress open --config-file cypress/integration/e2e/cypress-config.json",
+        "test:ui": "cypress open --config-file cypress/integration/ui/cypress-config.json",
+        "test:dev": "cypress run --env printLogsToConsole=always --config-file cypress/integration/rest-api/cypress-config.json",
+        "test:parallel": "cypress run --record --group apis --spec 'cypress/integration/rest-api/apis/**/*' --config-file cypress/integration/rest-api/cypress-config.json",
```

Comment:
The files and folders for these commands don't exist (yet?), so they won't work.